lm_eval/models/gpt2.py

```diff
@@ -3,6 +3,14 @@
 from lm_eval.base import BaseLM
 
 
+def get_args_for_accelerate(pretrained, max_memory):
+    config = transformers.AutoConfig.from_pretrained(pretrained)
+    max_memory = {i:max_memory for i in range(torch.cuda.device_count())}
+
+    torch_dtype = config.torch_dtype
```

Comment:
This is a good default but might be something the user wants to customize? Not familiar with this script so will let you decide.

Reply:
I think it is good if the user decides it too. Added it in 30b6b1bd54b0f406507cc1cfac69e76f4a0551f2